src/formatDate.js

```diff
@@ -8,7 +8,30 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const fromSeparator = fromFormat.pop();
+  const datePartsArray = date.split(fromSeparator);
```

Comment:
The function assumes that the date string will always match the `fromFormat`. Consider adding validation to ensure that the date string is in the expected format to prevent potential runtime errors.